Code/Editor/MainWindow.cpp

```diff
@@ -2137,6 +2160,291 @@ bool MainWindow::focusNextPrevChild(bool next)
+        actionProperties.m_name = "AWS Support";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.resources.awssupport", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnDocumentationAWSSupport();
+            }
+        );
+    }
+
+    // About O3DE
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "&About O3DE";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.abouto3de", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnAppAbout();
+            }
+        );
+    }
+
+    // Welcome
+    {
+        ActionProperties actionProperties;
+        actionProperties.m_name = "&Welcome";
+        actionProperties.m_category = "Help";
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.help.welcome", actionProperties,
+            [cryEdit]()
+            {
+                cryEdit->OnAppShowWelcomeScreen();
+            }
+        );
+    }
+
+}
+
+void MainWindow::InitializeMenus()
+{
+    // Initialize Menus
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&File";
+        m_menuManagerInterface->RegisterMenu(FileMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Edit";
+        m_menuManagerInterface->RegisterMenu(EditMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Game";
+        m_menuManagerInterface->RegisterMenu(GameMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Tools";
+        m_menuManagerInterface->RegisterMenu(ToolsMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&View";
+        m_menuManagerInterface->RegisterMenu(ViewMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "&Help";
+        m_menuManagerInterface->RegisterMenu(HelpMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "Documentation";
+        m_menuManagerInterface->RegisterMenu(HelpDocumentationMenuIdentifier, menuProperties);
+    }
+    {
+        MenuProperties menuProperties;
+        menuProperties.m_name = "GameDev Resources";
+        m_menuManagerInterface->RegisterMenu(HelpGameDevResourcesMenuIdentifier, menuProperties);
+    }
+
+    // Add Menus to MenuBar
+    QMenuBar* menuBar = this->menuBar();
+    menuBar->clear();
+    menuBar->addMenu(m_menuManagerInterface->GetMenu(FileMenuIdentifier));
```

Comment:
question: Maybe in future we want an interface to do this too? 🤔 Just a thought for adding a menu that isn't created here centrally

Reply:
Yes, I have a follow-up task to add MenuBar management. So it will be possible to add an entirely new menu to the File... section of the main window, for example. Didn't plan on it cause I thought it was just a menu, but since it's a different concept in Qt it will need a different API most likely.

Reply:
Got you, good plan 👍